pkg/kubelet/pluginmanager/pluginwatcher/plugin_watcher.go

```diff
@@ -17,35 +17,45 @@ limitations under the License.
 package pluginwatcher
 
 import (
+	"context"
 	"fmt"
 	"os"
 	"runtime"
 	"strings"
 	"time"
 
 	"github.com/fsnotify/fsnotify"
+	"k8s.io/client-go/util/flowcontrol"
 	"k8s.io/klog"
 
 	"k8s.io/kubernetes/pkg/kubelet/pluginmanager/cache"
 	"k8s.io/kubernetes/pkg/kubelet/util"
 	utilfs "k8s.io/kubernetes/pkg/util/filesystem"
 )
 
+const (
```

Comment:
I kind of just made these values up :) In my mind, they don't need to be super fine tuned ... we're mostly just focusing on preventing clear DOS situations. But definitely happy to hear any feedback.